.github/actionlint.yaml

```diff
@@ -16,7 +16,7 @@ self-hosted-runner:
     - linux.8xlarge.nvidia.gpu
     - linux.16xlarge.nvidia.gpu
     - linux.g5.4xlarge.nvidia.gpu
-    # Organization-wide AWS Linux Runners on Linux Foundation account
+    # Pytorch/pytorch AWS Linux Runners on Linux Foundation account
```

Comment:
These can't be used to run domain jobs ?

Reply:
Nope